template.yml.erb

```diff
@@ -225,12 +368,21 @@ Resources:
       FunctionName: !Ref StartSessionAndRelayMessagesFunction
       Principal: apigateway.amazonaws.com
 
-  WebsocketAuthorizerLambda:
+# AWS does not seem to be case sensitive, but is case aware. Therefore, we can't update from
+# Websocket here to WebSocket as is used elsewhere when updating an existing stack. Updating this is
+# a step we should take when we create our non-beta environment from scratch. At that point, we
```

Comment:
Should we create a tech debt task for this?

Reply:
Good call: https://codedotorg.atlassian.net/browse/JAVA-470